x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/components/agent_activity_flyout.tsx

```diff
@@ -92,12 +92,16 @@ export const AgentActivityFlyout: React.FunctionComponent<{
   const onClickViewAgents = async (action: ActionStatus) => {
     try {
       const { data } = await sendPostRetrieveAgentsByActions({ actionIds: [action.actionId] });
-
       if (data?.items?.length) {
-        const kuery = getKuery({ selectedAgentIds: data.items });
-        setSelectedStatus([]);
+        const kuery = getKuery({
+          selectedAgentIds: data.items,
+        });
         setSearch(kuery);
       }
+      if (action?.type === 'UNENROLL' || action?.type === 'FORCE_UNENROLL') {
+        setSelectedStatus(['unenrolled', 'inactive']);
```

Comment:
Does it mean that only `unenrolled` and `inactive` statuses are visible or all statuses? I'll try it locally. Ideally it should be all statuses, so agents are visible if unenroll is in progress. For simple unenroll, agents will move to `Offline` at first.

Reply:
It could be for all cases. Does the `Clear filters` button reset to default filters?

Reply:
That applies the `clearFIlters` function here: 
https://github.com/elastic/kibana/blob/662fd8572ef09b7092f794e140f6a2c042dfc07f/x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx#L116-L123

Reply:
I changed so that it makes all possible statuses visible for all cases. I think I had misunderstood how this filtering works.